translations/data/home.json

```diff
@@ -1,125 +1,143 @@
+	"de-de": "Unabhängig davon, für welchen E-Mail-Anbieter Du Dich entschieden hast, kannst Du an <mark>jeden</mark> auf der Welt E-Mails schicken, ohne Dich darum kümmern zu müssen, welchen Anbieter der Empfänger nutzt.",
         "sk": "Bez ohľadu na tvoju voľbu, môžeš poslať email <mark>hocikomu</mark> na celom svete.",
         "fr": "Indépendamment du choix que vous avez effectué, vous pouvez adresser un email à toute personne dans le monde, sans avoir à vous soucier du service qu'elle utilise.",
         "ru": "Независимо от вашего выбора, вы всегда можете отправить письмо <mark>любому</mark> человеку в мире, не зная и не думая о том, каким сервисом они пользуются.",
         "ar": "بغض النظر عن اختيارك ، يمكنك إرسال بريد إلكتروني إلى <mark> أي شخص </ mark> في العالم دون معرفة أو الاهتمام بالخدمة التي يستخدمونها." 
     },
     "intro_btn_neat": {
         "en-us": "Neat",
+	"de-de": "Prima - weiter",
         "sk": "Pekné",
         "fr": "Sympa !",
         "ru": "Неплохо",
         "ar": "أنيق"
     },
     "intro_not_social_media": {
         "en-us": "This is not the case for social media",
+	"de-de": "Das gilt aber nicht für Social Media.",
         "sk": "Sociálne média takto nefunguju.",
         "fr": "Ce n'est pas le cas pour les réseaux sociaux.",
         "ru": "С социальными сетями всё по-другому.",
         "ar": "هذا ليس هو الحال بالنسبة لوسائل الإعلام الاجتماعية"
     },
     "intro_social_media": {
         "en-us": "If you're on Instagram or TikTok, and your friends or celebrities you like are on Twitter, or Facebook, you will have to sign up for <mark>each separate service</mark> to stay in touch with them.",
+	"de-de": "Wenn Du bei Instagram oder TikTok bist, aber Deine Freunde oder Prominente bei Twitter oder Facebook sind, musst Du Dich bei <mark>jedem</mark> dieser Anbieter registrieren, um mit ihnen in Kontakt zu treten.",
         "sk": "Ak si na Instagrame alebo TikToku a tvoji priatelia alebo celebrity, o ktoré sa zaujímaš sú na Twitteri alebo Facebooku, musíš sa zaregistrovať <mark>na každej sociálnej sieti</mark>.",
         "fr": "Si vous êtes sur Instagram ou TikTok, et vos amis, ou les célébrités que vous suivez sont sur Twitter ou Facebook, vous devrez créer un compte sur <mark>chacun de ces services séparément</mark> pour rester en contact avec eux.",
         "ru": "Если вы пользуетесь Instagram или TikTok, а ваши друзья или знаменитости, которые вам нравятся, используют Twitter, или Facebook, вам придётся зарегистрироваться на <mark>каждом отдельном сервисе</mark>, чтобы с ними держать связь.",
         "ar": "إذا كنت تستخدم Instagram أو TikTok ، وكان أصدقاؤك أو المشاهير الذين تحبهم على Twitter أو Facebook ، فسيتعين عليك الاشتراك في <mark> كل خدمة منفصلة </ mark> للبقاء على اتصال معهم." 
     },
     "intro_btn_good_point": {
         "en-us": "Good point",
+	"de-de": "Stimmt - gutes Argument !",

```

Comment:
Also here, something feels to much to me. Maybe just "Gutes Argument" is enough?

Reply:
Yes, let's keep these labels short.